marklogic-client-api/src/test/java/com/marklogic/client/test/rows/ValidateDocTest.java

```diff
@@ -51,7 +54,10 @@ public void moreSetup(){
     }
 // TODO : test with rest-writer user when https://bugtrack.marklogic.com/57978 is fixed
```

Comment:
We don't want to change the user, we'll just remove the "xdmp-invoke" privilege from the "test-rest-writer" role.